go/signedexchange/structuredheader/parser.go

```diff
@@ -138,7 +183,7 @@ func (p *parser) parseParameterisedIdentifier() (ParameterisedIdentifier, error)
 		if _, ok := parameters[param_name]; ok {
 			return ParameterisedIdentifier{}, fmt.Errorf("structuredheader: duplicated parameter '%s'", param_name)
 		}
-		var param_value interface{}
+		var param_value Item
```

Comment:
subCamelCase (for other vars too. sorry, should have caught in earlier reviews)

Reply:
Oops, done.